services/crypto.proto

```diff
@@ -0,0 +1,6 @@
+message Roster {
```

Comment:
Multiple comments on the protofiles:
 + the ServerIdentity must also be generated
 + I'm not sure about putting that as "crypto.proto", it's only a list of server, there's not really any crypto.
 + I wonder what's the best place to put that also. Should all proto files should be placed under the package where they are defined (in that case SDA) ? Or all in a single package ? I'm not sure about defining Roster in the Service package, which are un-related. but again, I don't know the "best practices"... What do you think ?